wdl2cwl/main.py

```diff
@@ -91,6 +91,38 @@ def get_command(
     return new_command
 
 
+def get_input(
+    inputs: List[cwl.CommandInputParameter],
+    unbound_input: List[str],
+    bound_input: List[str],
+) -> List[cwl.CommandInputParameter]:
+    """Get bound and unbound inputs."""
+    for i in unbound_input:
+        input_type = wdl_type[i[0]]
+        input_name = i[1]
+        inputs.append(cwl.CommandInputParameter(id=input_name, type=input_type))
+
+    for i in bound_input:
+        input_type = wdl_type[i[0]]
+        input_name = i[1]
+        input_value: Any = i[2].replace('"', "")
```

Comment:
Why `Any` here?

Reply:
When I run tox I got an incompatible type assignment error, because the value of input_value would later be changed to boolean and int.

Reply:
That's fine, you can notate the multiple types it can be like this

```suggestion
        input_value: Union[str, bool, int] = i[2].replace('"', "")
```

Reply:
```
wdl2cwl/main.py:113:32: error: Item "bool" of "Union[str, bool, int]" has no attribute "lower"  [union-attr]
                input_value = bool(input_value.lower() == "true")
                                   ^
wdl2cwl/main.py:113:32: error: Item "int" of "Union[str, bool, int]" has no attribute "lower"  [union-attr]
                input_value = bool(input_value.lower() == "true")
```
Now it gives this error. Shall I switch back to Any?

Reply:
There are several possible solutions. The simplest may be to have a `raw_input_value` of the type `str` on line 110 and an `input_value` of type `Union[str, bool, int]` used for the lines 112-115 section, turned that section into an `if` boolean, `elif` int, `elif` string, `else` throw an exception block.